codegen/src/main/java/software/amazon/awssdk/codegen/naming/DefaultNamingStrategy.java

```diff
@@ -411,6 +412,12 @@ public String getUnionEnumTypeName(MemberModel memberModel) {
         return screamCase(memberModel.getName());
     }
 
+    @Override
+    public String getUniqueEventStreamEventShapeName(Member eventMember, String eventStreamName) {
+        return eventMember.getShape() + eventStreamName;
```

Comment:
Just checking: is it necessary to have `eventmember.getShape()` be capitalized here?

Reply:
What do you mean? The parameter name `eventMember`? Or do you mean we should apply some naming transform of the shape name to ensure it has correct capitalization?

RE: the parameter name, theres no reason it couldn't be `eventmember`.

RE: applying capitalization to the existing shape name - Thats a good question - I was assuming that the existing shape name has already been validated (and any divergence from standard is intended and should be preserved).